content/docs/command-reference/plots/modify.md

```diff
@@ -123,6 +123,29 @@ plots:
       title: Accuracy
 ```
 
+## Example: Make a regular output into a plot
+
+We can use `dvc plots modify` to make an <abbr>output</abbr> or metrics file of
```

Comment:
metrics file > simple metrics file

and link to https://dvc.org/doc/command-reference/metrics

Reply:
Missing link part.

Reply:
Sorry. Missed the link. Added now.